change-notes/1.26/analysis-javascript.md

```diff
@@ -27,7 +31,11 @@
   - [needle](https://www.npmjs.com/package/needle)
   - [object-inspect](https://www.npmjs.com/package/object-inspect)
   - [pretty-format](https://www.npmjs.com/package/pretty-format)
+  - [react](https://www.npmjs.com/package/react)
+  - [react-router-dom](https://www.npmjs.com/package/react-router-dom)
+  - [react-redux](https://www.npmjs.com/package/react-redux)
```

Comment:
```suggestion
  - [react-redux](https://www.npmjs.com/package/react-redux)
  - [redis](https://www.npmjs.com/package/redis)
  - [redux](https://www.npmjs.com/package/redux)
```

You were missing `redux` from the list.   
(And I snuck in `redis`, which I forgot in https://github.com/github/codeql/pull/4559).